ko/lessons/specifics/ecto.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: page
+title: Ecto
+category: specifics
+order: 2
+lang: ko
+---
+
+Ecto는 공식적인 Elixir 프로젝트로 데이터베이스를 감싸는 부분과 종합적인 질의 언어를 제공합니다. Ecto를 사용하면 마이그레이션의 생성과 모델의 정의, 레코드의 추가와 삭제, 그리고 질의를 할 수 있게 됩니다.
+
+{% include toc.html %}
+
+## 설치하기
+
+우선 Ecto와 데이터베이스 어댑터를 프로젝트의 `mix.exs`에 추가해야 합니다. 대응하고 있는 데이터베이스 어댑터의 목록은 Exto의 README 에 있는 [Usage](https://github.com/elixir-lang/ecto/blob/master/README.md#usage)에서 확인할 수 있습니다. 이 예제에서는 PostgreSQL을 사용합니다:
```

Comment:
[typo] 우선 Ecto와 데이터베이스 어댑터를 프로젝트의 `mix.exs`에 추가해야 합니다. 대응하고 있는 데이터베이스 어댑터의 목록은 **Exto**(->Ecto)의 README 에 있는 [Usage](https://github.com/elixir-lang/ecto/blob/master/README.md#usage)에서 확인할 수 있습니다. 이 예제에서는 PostgreSQL을 사용합니다:

And, I think `대응되는` or `지원되는` is proper rather than `대응하고 있는`